website/pages/gamejam_en.md

```diff
@@ -3,7 +3,36 @@
 Get ready for our upcoming event, Global Game Jam!
 This year's edition of Global Game Jam will be held 29th - 31st January.
 Because COVID-19 is still wreaking havoc in the world, the jam will be held remotely in our [Discord server](https://discord.gg/eHgXYMS). Also check out our [Facebook event](https://fb.me/e/1SCmIAaRH).
-Join our site on Global Game Jams official [website](https://globalgamejam.org/2021/jam-sites/liu-game-jam)!
+
+**To participate**
```

Comment:
Om vi ska snacka WEB2.0, kanske man ska använda en heading istället för att göra det bold?
`### To Participate`

Bör också vara stor bokstav på "Participate" om det är en engelsk titel! :D